src/main/java/com/rocketden/main/model/Room.java

```diff
@@ -0,0 +1,24 @@
+package com.rocketden.main.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+public class Room {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+
+    private String roomId;
+
+    private LocalDateTime createdDate = LocalDateTime.now();
```

Comment:
Minor detail but would this field make more sense to be named as `createdDateTime` since it also includes time and not just date?

Reply:
Yeah that makes sense, just updated it and will merge now 👍